presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/H2ResourceGroupsModule.java

```diff
@@ -35,10 +38,12 @@
     @Override
     public void configure(Binder binder)
     {
-        configBinder(binder).bindConfig(DbResourceGroupConfig.class);
+        configBinder(binder).bindConfig(DbReloadingResourceGroupConfig.class);
+        configBinder(binder).bindConfig(ReloadingResourceGroupConfig.class);
         binder.bind(ResourceGroupsDao.class).toProvider(H2DaoProvider.class).in(Scopes.SINGLETON);
-        binder.bind(DbResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
-        binder.bind(ResourceGroupConfigurationManager.class).to(DbResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
+        binder.bind(ReloadingResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
+        binder.bind(ManagerSpecProvider.class).to(DbManagerSpecProvider.class).in(Scopes.SINGLETON);
+        binder.bind(ResourceGroupConfigurationManager.class).to(ReloadingResourceGroupConfigurationManager.class).in(Scopes.SINGLETON);
```

Comment:
As I have already commented in my previous comments on this PR - `ReloadingResourceGroupConfigurationManager` should be binded in `ReloadingResourceGroupConfigurationModule` and you should just install that instead of binding this class in every module